fplc_maintenance_scheduler.py

```
@@ -0,0 +1,111 @@
+
+class MaintenanceScheduler:
+    def __init__(self):
+        self.maintenance_tasks = {
+            'pump_seals': {'interval': 180, 'last_done': None},
+            'detector_calibration': {'interval': 90, 'last_done': None},
+            'system_cleaning': {'interval': 30, 'last_done': None}
+        }
+    
+    def check_maintenance_due(self):
+        due_tasks = []
+        for task, info in self.maintenance_tasks.items():
```

Ruff (0.12.2)

Loop control variable `info` not used within loop body

Rename unused `info` to `_info`

(B007)